aws_lambda_builders/workflows/java/utils.py

```diff
@@ -37,17 +40,8 @@ def exists(self, p):
     def which(self, executable, executable_search_paths=None):
         return which(executable, executable_search_paths=executable_search_paths)
 
-    def copytree(self, source, destination):
-        if not os.path.exists(destination):
-            self.makedirs(destination)
-        names = self.listdir(source)
-        for name in names:
-            new_source = os.path.join(source, name)
-            new_destination = os.path.join(destination, name)
-            if os.path.isdir(new_source):
-                self.copytree(new_source, new_destination)
-            else:
-                self.copy(new_source, new_destination)
+    def copytree(self, source, destination, ignore=None, include=None):
```

Comment:
Why do we need this copytree instead of using aws_lambda_builders.utils.copytree in the callers?

Reply:
We used to have a custom method here to copy the files and folders. I've changed here to use the one in `aws_lambda_builders.utils.copytree`. I didn't want to change every place which uses this copy in order to keep the change in a meaningful scope. 
But I agree that we can do some code cleanup or refactoring some parts of lambda builders in the future. There are lots of duplication and multiple implementation of the same logic.